### PR TITLE
st1 [658][IMP] Implement queue_job on order confirmation email

### PR DESCRIPTION
**Pull request by TimLai125**
[658](https://www.quartile.co/web#id=658&action=771&model=project.task&view_type=form&menu_id=505)

### CODE REVIEW

File: website_sale_adj/models/sale_order.py [hunk 1]
```diff
@@ -86,21 +88,23 @@ def force_quotation_send(self):
...
                 email_ctx["email_bcc"] = ",".join(
                     [str(partner.email) for partner in order.message_partner_ids]
                 )
                 email_ctx["reply_to"] = order.message_get_reply_to(
                     [order.id], default=email_ctx["default_email_from"]
                 )[order.id]
-                self.env["mail.template"].browse(
-                    email_ctx.get("default_template_id")
-                ).with_context(email_ctx).send_mail(order.id)
+                order.with_delay().send_confirmation_email(email_ctx)
```
Comment (yostashiro) [inline website_sale_adj/models/sale_order.py]: @TimLai125 Can we add a description to the job, please?

Comment (TimLai125) [reply website_sale_adj/models/sale_order.py]: You mean passing an argument to the job to set with meaningful job name or a comment for this line?

Comment (yostashiro) [reply website_sale_adj/models/sale_order.py]: @TimLai125 The former.

**Review by yostashiro — APPROVED** [review]
LGTM 👍

**Pull request merged**